Update interfaces/css-layout-api.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Note: This file was recently manually updated in commit 876f188904.
      This commit may revert some of those changes.

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/css-layout-api.idl
Build: https://travis-ci.org/w3c/webref/builds/182613166

stephenmcgruer commented:
This doesn't seem to have an idlharness.html file (noting that no tests were affected by this change). We should add one, otherwise this idl file is pointless.

autofoolip committed:
Update interfaces/css-layout-api.idl

Note: This file was recently manually updated in commit 876f188904.
      This commit may revert some of those changes.

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/css-layout-api.idl
Build: https://travis-ci.org/w3c/webref/builds/182613166
